public/Lab-Files/Lesson03/Challenges/wild-animal.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Wild Animal</title>
+    <style>
+img {
+    display: block;
+    margin: 0 auto;
+}
+</style>
+</head>
+<body style="background-color:#f9a743;">
+
+    <h1 style="text-align:center; color:#3d2810;">The coolest creature</h1>
+<img src="lions.jpeg" alt="Lion laying in the grass">
+<p style="text-align:center; color:#8e5a1d"> The <em>Lion</em> is awesome because it is one of the <strong>strongest and most powerful</strong> of all the wild creatures. Lions are known to project a <strong>regal and majestic</strong> quality when hunting and caring for their young. They are native to the <strong>sub-Saharan African Desert</strong> and are the the only large cats to travel in groups called "<strong>Prides</strong>." Lions are known to be very <strong>affectionate</strong> towards one another and also very <strong>expressive</strong> through their physical movements and behaviors. Lions are extaordinary animals that symbolize <strong>strength</strong> and <strong>beauty</strong> and I will always admire them.</p>
+</body>
+
+</html>
```

Comment:
Looks great. The only thing I suggest is adding all the CSS styles into the head of your document. The inline styles were not necessary. 
Great job though, you did good overall!